homeassistant/components/lifx/coordinator.py

```diff
@@ -205,13 +205,20 @@ async def _async_populate_device_info(self) -> None:
             methods, DEFAULT_ATTEMPTS, OVERALL_TIMEOUT
         )
 
+    def get_number_of_zones(self) -> int:
+        """Return the number of zones.
+
+        If the number of zones is not yet populated, return 0
+        """
+        return len(self.device.color_zones) if self.device.color_zones else 0
```

Comment:
There is always 1 zone, even on non-multizone devices.

Reply:
https://github.com/home-assistant/core/pull/92683